### DISCUSSION TITLE
More types of field in Profile Management

### DISCUSSION THREAD

**Discussion by dtquocbao** [oqtane/oqtane.framework#821] [Q&A] [answered]
Hello folks,

I am working on the Profile Management module for my little app and just found that seems Oqtane has currently support text field only and not with the other such as dropdownlist, checkbox... E.g.: Status (Single/Window/Married)

Any idea or solution can you share please? Since I suppose that modify core code is not a good idea.

Thank you

**Comment by sbwalker** [comment, marked as answer]
Currently profile management is extensible in the sense that you can add as many profile fields as you want. However you are correct that it is limited in the types of user interactions. If you do not want to modify the core code you can create your own Profile module with enhancements and replace the default module on the Profile page.

**Reply by dtquocbao** [reply]
Thanks @sbwalker

**Reply by sbwalker** [reply]
@rockydant there is now an "Options" field in Profile Management which allows you to specify a comma delimited list of options for a profile item. You can also use the existing Default Value field to set the default option. See https://github.com/oqtane/oqtane.framework/pull/843/commits/0ef04e81ff10945dcbc129a27776ce3d6bf8bead

**Reply by dtquocbao** [reply]
Tha's cool @sbwalker!

Thanks so much for your quickly support!
